addons/pos_restaurant/static/src/js/PointOfSaleModel.js

```diff
@@ -106,6 +106,9 @@ odoo.define('pos_restaurant.PointOfSaleModel', function (require) {
                 const table = this.getRecord('restaurant.table', order.table_id);
                 this.data.uiState.activeTableId = order.table_id;
                 this.data.uiState.activeFloorId = table.floor_id;
+
+                if(order.lines)
```

Comment:
Cool, I didn't think to solve it this way. Very nice. But can you put another condition such that we only select an orderline if there is no selected orderline?